cmd/eventsprinter/logic/logic.go

```diff
@@ -33,11 +33,6 @@ func PrintEvents(url, topic, subscription string, verbose bool) error {
 			}
 			consumer.Ack(msg)
 
-			// We're only interested in control messages.
```

Comment:
I've removed the concept of "control messages" as it dates from a time when we were going to keep utilisation messages separate.  As we no longer do this, it's not being used for anything.